internal/ctw/utils.go

```diff
@@ -3,7 +3,8 @@ package ctw
 import "strings"
 
 var (
-	white      string = "\033[38;2;255;255;255m"
+	noColor    string = "\033[0m"
+	white      string = "\033[38;2;255;255;255m" // not used anymore
```

Comment:
No need of keeping the `white` var around. It would be great if you can update the PR and while at it, consider replacing the reference to `white` with `noColor` in `DisplayColor` function (line 15 of `internal/ctw/utils.go`)

Reply:
I don’t have so much time to do these changes. If you or someone else could do that, it would be great.